### PR TITLE
Add IRM gas tests

### PR DESCRIPTION
**Pull request by Rubilmax**

### CODE REVIEW

**Review by MerlinEgalite — APPROVED** [review]

**Comment by Rubilmax** [comment]
> to update

once IRM is merged and finalized and after all fixes, this pr is not a priority

**Review by Jean-Grimal — APPROVED** [review]

File: package.json
Comment (QGarchery) [inline package.json:1]: Is the packages update needed ? Not blocking, just curious

File: package.json [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
   ],
   "scripts": {
-    "prepare": "husky install",
-    "postinstall": "forge install",
+    "prepare": "husky install && forge install",
     "build:forge": "FOUNDRY_PROFILE=build forge build",
-    "build:hardhat": "npx hardhat compile",
+    "build:hardhat": "hardhat compile",
     "build:blue": "cd lib/morpho-blue/ && yarn build:forge && cd ../..",
+    "build:irm": "cd lib/morpho-blue-irm/ && yarn build:forge && cd ../..",
```
Comment (QGarchery) [inline package.json:16]:
```suggestion
    "build:irm": "yarn --cwd lib/morpho-blue-irm build:forge",
```
[optional]

**Review by QGarchery — APPROVED** [review]

File: package.json
Comment (Rubilmax) [reply package.json:1]: No it is not, it is mostly about having the latest versions but that may not be a good practice

**Pull request merged**